.github/workflows/translations-sync.yml

```
@@ -115,18 +115,18 @@ jobs:
 
       - name: Patch version if the files changed
         run: |
-          CHANGED_FILES=$(git diff --name-only origin/main HEAD -- packages/website-i18n)
+          CHANGED_FILES=$(git diff --name-only $GITHUB_SHA HEAD -- packages/website-i18n)
```

```suggestion
          CHANGED_FILES=$(git diff --name-only "$GITHUB_SHA" HEAD -- packages/website-i18n)
```
I don't think this could be abused, but quotes for safety

---

Can you make this suggestion on #8118?